Changes.md

```diff
@@ -26,6 +26,7 @@ Improvements
   - <kbd>Shift+Alt+E</kbd> : Edit tweaks node.
 - Edit Menu : Added item to rename one or more selected nodes, with accompanying keyboard shortcut (#4286).
 - Box : Rename plug dialogue now ensures names are valid as they are typed.
+- ShaderQuery and ShaderTweaks : Improved labels of menu items that open shader browsers by adding "..." to the item name.
```

Comment:
This needs to go in the new `0.61.x.x` section at the top. I feel like we need a better way of managing Changes.md to make it easier to get right, but so far ideas (at least simple ones) have eluded me...

Reply:
Ah yes, of course. Fixed in ad20bccd80fbcc7783a289a3597fd85bc2358345